simulator/programming/index.md

```diff
@@ -22,11 +22,28 @@ On first run, the robot will execute an example program for convenience. This pr
 
 Your code should be developed in `robot.py`.
 
-
 <div class="warning">
   Only your controller code will be present in the competition environment.
 </div>
 
+### Running multiple robots
+
+To test how your robot behaves in each starting zone of the arena, you can copy your robot's code to run in each corner. Code can be placed in a `zone-<zone>` directory to run in starting zone `<zone>`:
+
+```
+.
+├── competition-simulator-<version>
+│   ├── ...
+│   └─ worlds
+│       └── Arena.wbt
+├── zone-0
+│   └── robot.py
+└── zone-1
+   └── robot.py
+```
+
+This will run 2 robots in the arena, each with different `robot.py`s. You can run as many or as few zones as you like, in any combination.
```

Comment:
```suggestion
This will run two robots in the arena, each with different `robot.py`s. You can run as many or as few zones as you like, in any combination.
```